config/puma.rb

```diff
@@ -0,0 +1,15 @@
+workers Integer(ENV["WEB_CONCURRENCY"] || 2)
+threads_count = Integer(ENV["MAX_THREADS"] || 5)
+threads threads_count, threads_count
+
+preload_app!
+
+rackup      DefaultRackup
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.